src/app/(proper_react)/redesign/(authenticated)/user/welcome/scanning/page.tsx

```diff
@@ -0,0 +1,123 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextRequest } from "next/server";
+import { getL10n } from "../../../../../../functions/server/l10n";
+import {
+  getOnerepProfileId,
+  getLatestOnerepScan,
+  setOnerepScan,
+  setOnerepScanResults,
+} from "../../../../../../../db/tables/subscribers";
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../../../../../api/utils/auth";
+
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: "Welcome - Scanning", // FIXME l10n.getString("breach-meta-title"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: process.env.SERVER_URL,
+      images: ["/images/og-image.webp"],
+    },
+  };
+}
+
+export default async function UserWelcomeScanning() {
+  const session = await getServerSession(authOptions);
+
+  const current = 0;
+  const total = 672;
+  const percentage = ((current / total) * 100).toFixed(1);
+
+  async function callOneRep(method: string, path: string) {
+    const bearerToken = process.env.ONEREP_API_KEY;
+    const options = {
+      method,
+      headers: {
+        Authorization: `Bearer ${bearerToken}`,
+        "Content-Type": "application/json",
+      },
+    };
+    if (method !== "GET" && method !== "HEAD") {
+      //@ts-ignore FIXME
+      // options.body = body;
+    }
+    const result = await fetch(
+      `${process.env.ONEREP_API_BASE}/${path}`,
+      options
+    );
+    if (!result.ok) {
+      throw new Error("Error connecting to provider");
+    }
+    return result.json();
+  }
+
+  // @ts-ignore FIXME
+  const result = await getOnerepProfileId(session?.user?.subscriber?.id);
```

Comment:
And while my special 👁️'s are here, re: abundance of `?.` operators does this mean we could be calling `getOnerepProfileId(undefined)` and we should avoid that if the user is not a subscriber (or logged in, or whatever)?

Also, a bit of a nit but this should probably be "OneRep", per https://onerep.com/ (ie: "© 2023 OneRep LLC").
But I think that code is probably outside the scope of this PR, but I'm a stickler for style guides.

We call it `callOneRep()` above (L44), but also `*Onerep*` in these imports (from the /db/tables/subscribers table):

https://github.com/mozilla/blurts-server/blob/911c5ec128440278cd90e66f9d37e719374080f8/src/app/(proper_react)/redesign/(authenticated)/user/welcome/scanning/page.tsx#L7-L12

```sh
git grep -n "Onerep" src # 22 results
```

Although I think almost all instances of "Onerep" in the codebase referred to the functions/exports from [src/db/tables/subscribers.js exports](https://github.com/mozilla/blurts-server/blob/911c5ec128440278cd90e66f9d37e719374080f8/src/db/tables/subscribers.js#L475-L493):

```js
476:  getLatestOnerepScan,
477:  getOnerepProfileId,
...
491:  setOnerepProfileId,
492:  setOnerepScan,
493:  setOnerepScanResults,
```

Reply:
This is bugging me too, might as well make it consistent now. Thanks!

Reply:
FYI I put some guards in to ensure that users have a session and a bearer token and exit early if not, but this is likely unnecessary since this is all under an authenticated route. I don't really understand Next deeply enough to know what's safe, so I am leaning on the "duplicate checks" side.